packages/cmf/src/bootstrap.md

```diff
@@ -37,3 +37,18 @@ cmf.bootstrap({
 | middlewares | Array | undefined | Redux middlewares |
 | storeCallback | function | undefined | Let you call a function once the store is created |
 | AppLoader | React Component | undefined | Let you define the React component to use to show the app is currently loading (waiting for the settings) |
+
+## Modules
+
+You apps may be splitted into different modules, like a core set of components and some other business logics.
+
+For that CMF expose an API to let you merge your settings:
+
+```javascript
+import cmf from '@talend/react-cmf';
+
+import containersModule from '@talend/react-containers/lib/cmfModule';
+import myAppModule from './cmfModule';
+
+cmf.boostrap(cmf.modules.merge(containersModule, myAppModule));
```

Comment:
I would see another api instead : 
```
import dataset from '@talend/dataset';
import sharing from '@talend/sharing';

cmf.boostrap({
    ...myAppModule,
    modules: [dataset, sharing]
});
```

And it's cmf.boostrap that merges : 1 less call, 1 less complexity

Reply:
I really like that, no more things exported, lets make cmf a set of tools + bootstrap and it s ok.